src/core/core.controller.js

```diff
@@ -147,8 +147,9 @@ function updateConfig(chart) {
 	chart.tooltip.initialize();
 }
 
-function positionIsHorizontal(position) {
-	return position === 'top' || position === 'bottom';
+const WELL_KNOWN_POSITIONS = ['top', 'bottom', 'left', 'right', 'chartArea'];
+function positionIsHorizontal(position, axis) {
```

Comment:
can we just do `axis === 'x'` instead?

Reply:
We might now that I removed default positions. Before I did that, it would override the fact that the position was wrong and the config would not be updated.

Reply:
Tried this. It breaks the default position fixing because it overrides the wrong axis position value. Lets leave as is for now